content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+
+```yaml
+---
+api_version: core/v2
+type: CheckConfig
+metadata:
+  namespace: default
+  name: webservers-aggregate-check
+spec:
+  runtime_assets:
+  - sensu/sensu-aggregate-check
+  command: sensu-aggregate-check --api-user=foo --api-pass=bar --entity-labels='server_role:webserver' --warn-percent=75 --crit-percent=50
+  subscriptions:
+  - backend
+  publish: true
+  interval: 30
+  handlers:
+  - slack
+  - pagerduty
+  - email
+```
+
+The check command uses a username/password combination to access the API and then matches events with the label "server_role: webserver". In this case, the check will create an event if 75 percent of the aggregate events are in a `warning` state and will also create an event if 50 percent of the aggregate events are in a critical state.
+
+## Check Aggregates
+
+In addition to having entities comprise aggregates, checks can also comprise aggregates. Continuing with our scenario, let's suppose that our webservers are serving various applications on different ports: 80, 8080, 9000. A standard check grouping might look like:
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-80
+  namespace: default
+spec:
+  command: "check-http.rb -u http://webserver01.example.com"
+  handlers: 
+  - slack
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-8080
+  namespace: default
+spec:
+  command: "check-http.rb -u --port 8080 http://webserver01.example.com"
+  handlers: 
+  - slack
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-9000
+  namespace: default
+spec:
+  command: "check-http.rb -u --port 9000 http://webserver01.example.com"
+  handlers: 
+  - slack
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+So we have three separate checks that are monitoring our web application. However, if we wanted to view the webapp's health these three checks don't do the best job providing that insight. The checks are isolated from each other and each check alerts individually. 
+
+Instead, it makes more sense to have this group of checks be part of an aggregate--we might not care if a check on an individual host fails, but we certainly care if a large percentage of the checks are in a warning or critical state across a number of hosts.We can do this by adding a label to the checks:
```

Comment:
```suggestion
Instead, it makes more sense to configure this group of checks as an aggregate because you might not care if a check on an individual host fails, but you will certainly care if a large percentage of the checks are in a warning or critical state across a number of hosts.

To turn these checks into an aggregate, add a label to each of them:
```